OrbitCore/CaptureData.h

```diff
@@ -11,6 +11,8 @@
 #include "absl/container/flat_hash_map.h"
 #include "capture_data.pb.h"
 
+using orbit_client_protos::FunctionStats;
```

Comment:
Do we want to use `using` in header files or should we use the fully qualified name?

Reply:
I'm fine with both.

Reply:
it feels not right using using in header files tbh

Reply:
Okay, I remved the using from the header.